components/code_preview.jsx

```diff
@@ -129,17 +118,11 @@ export default class CodePreview extends React.PureComponent {
                 <span className='post-code__language'>
                     {`${this.props.fileInfo.name} - ${language}`}
                 </span>
-                <div className='post-code__container'>
-                    <code className='hljs'>
-                        <table>
-                            <tbody>
-                                <tr>
-                                    <td className='post-code__lineno'>{strlines}</td>
```

Comment:
if we are not using this class anywhere else, we should remove it from CSS as well

Reply:
Good catch. I thought I'd removed that, but obviously, I didn't.